thunder/executors/cudnn_layernormex.py

```diff
@@ -1,5 +1,6 @@
-from typing import Any
+from typing import Any, List
```

Comment:
doesn't seem to be used
```suggestion
from typing import Any
```

Reply:
Seems like pre-commit changed the type hint from `List` to `list`.
[[Here](https://github.com/Lightning-AI/lightning-thunder/pull/234/commits/0250461e8372544a526f2983dd67972fbcf648f1#diff-0729becfc13c06019a75178ac7afe4cebf0382a709b88fd932ac65fc5644a181R167)]